modules/rosetta/src/main/scala/org/tessellation/rosetta/domain/error/RosettaError.scala

```diff
@@ -0,0 +1,54 @@
+package org.tessellation.rosetta.domain.error
+
+import cats.syntax.option._
+
+import derevo.circe.magnolia.encoder
+import derevo.derive
+import enumeratum.values._
+import io.circe.Encoder
+
+@derive(encoder)
```

Comment:
Regular `encoder` will be fine? I would suggest using in the whole scope of rosetta the same `encoder`s and `decoders`, even if the json look the same.

Reply:
Good catch. The whole scope should use the same one